mllib/src/test/scala/org/apache/spark/ml/feature/Word2VecSuite.scala

```diff
@@ -144,6 +144,31 @@ class Word2VecSuite extends SparkFunSuite with MLlibTestSparkContext with Defaul
     }
   }
 
+  test("findSynonymsArray") {
```

Comment:
Can you please combine this with the findSynonyms test to avoid fitting the same model twice and to share code?